ftosv/README.md

```diff
@@ -0,0 +1,22 @@
+# Dell FTOSv (OS10) / ftosv
+
+This is the vrnetlab docker image for Dell FTOS10 virtual switch.
+
+
+## Building the docker image
+
+Put the .qcow2 file in this directory and run make docker-image and you should be good to go. The resulting image is 
+called vr-ftosv. You can tag it with something else if you want, like my-repo.example.com/vr-ftosv and then push it to 
+your repo. The tag is the same as the version of the FTOS image, so if you have dellftos.10.5.2.4.qcow2 your final docker 
+image will be called vr-ftosv:10.5.2.4
+
+NOTE:
+* Dell officially does not provide .qcow2 disk images. Check Dell OS10 virtualization documentation on how to prepare disk image from officially available virtualization package. One can use either GNS3 or EVE-NG to prepare .qcow2 disk.
```

Comment:
Hi @log1cb0mb 
when you refer to gns/eve for preparation steps, what are those steps exactly?

Reply:
@hellt Here is official guide from Dell for GNS3: https://cld.pt/dl/download/2f61626b-f1fe-4415-8825-4357823e5ed8/os10_virtualization_guide.pdf
And for EVE-NG, here is my MR where I added template support with instructions on preparing image: https://gitlab.com/eve-ng-dev/templates/-/merge_requests/3

Reply:
thanks, I think these prep steps (from the MR you did for eve) can be copied over here, as they will fit will.

Also, since dell uses the installation step, some images in vrnetlab pack this as part of the `make install` step. Meaning that when you do `make`, it first calls the `install` function which take the vmdks, convert them if needed, then start the node, wait for it to boot and commit the resulting image. 
The net positive effect is that you call `make` once, and you get the image that is already installed and ready to be used.

You can check an example of this approach here https://github.com/hellt/vrnetlab/blob/fa8d978723ccbc5de344a0c73cc3d9d032353d06/vmx/docker/launch.py#L418

but I am happy to merge this in its current form once you add the preparation steps to the README, and you can decide if you want to improve the support for ftos with embedded installation later on

Reply:
Yes, that is something I am already considering doing. This would certainly be nice touch and remove any external dependencies. If this is needed to be merged for containerlab build (I suppose not) then go ahead otherwise we can leave this open and I will work on including process of creating base image from source.

Reply:
sure, then let me merge this as it is